### PR TITLE
Add zenodo doi

### PR DESCRIPTION
**Pull request by jlaehne**
### Description of the change
Add citing section with zenodo doi and credits section to main page of user guide.

### Progress of the PR
- [X] Change implemented (can be split into several points),
- [X] update user guide (if appropriate),
- [x] add a changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/rosettasciio/blob/main/upcoming_changes/README.rst)),
- [x] Check formatting of the changelog entry (and eventual user guide changes) in the `docs/readthedocs.org:rosettasciio` build of this PR (link in github checks)
- [x] ready for review.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -36,6 +36,28 @@ Similarly, it takes a dictionary as input for file writers.
...
+find it by clicking on the DOI badge:
+
+.. image:: https://zenodo.org/badge/doi/10.5281/zenodo.8011666.svg
+   :target: https://doi.org/10.5281/zenodo.8011666
+
+
+Credits
+=======
+
+RosettaSciIO is maintened by `an active community of developers
```
Comment (pietsjoh) [inline docs/index.rst:57]: maintained

**Review by pietsjoh — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 1 above)
Comment (ericpre) [inline docs/index.rst:57]:
```suggestion
RosettaSciIO is developed by `an active community of developers
```

**Review by ericpre — APPROVED** [review]

**Comment by jlaehne** [comment]
Thanks @pietsjoh for catching this copy-pasted typo from HyperSpy, I submitted a PR to fix it there as well: https://github.com/hyperspy/hyperspy/pull/3211

@ericpre - indeed RSciIO is more in a development state ... but "developed by ... developers" is not the nicest phrasing either? So I am considering to stick with "maintained".

**Comment by ericpre** [comment]
Would "RosettaSciIO is developed by an active community of contributors" be better?

**Pull request merged**